src/actions/tests/sources.spec.js

```diff
@@ -131,8 +133,8 @@ describe("sources", () => {
     const fooSource = getSource(getState(), "foobar.js");
     const barSource = getSource(getState(), "barfoo.js");
 
-    expect(fooSource.get("text").indexOf("return 2")).not.toBe(-1);
-    expect(barSource.get("text").indexOf("return 3")).not.toBe(-1);
+    expect(fooSource.get("text").indexOf("return foobar.js")).not.toBe(-1);
```

Comment:
text is a string? if it is maybe `includes` would be clearer

Reply:
agreed, but lets do this later :)